### ISSUE TITLE
Actionlint outdated, doesn't allow 'id-token' permissions.

### ISSUE THREAD

**Issue by dan-ih** [github/super-linter#2063]
## Describe the bug

The id-token permission is being flagged by the linter as invalid. id-token is now a valid permission in GHA. 

This is fixed in [actionlint v1.6.6](https://github.com/rhysd/actionlint/releases/tag/v1.6.6)

A re-release of superlinter should pick up the latest actionlint and resolve this issue.

## Expected behavior

The 'id-token' permission should be able to be set without having the linter fail

## Steps to Reproduce
1. Add the following permission to any github action workflow:
```
    permissions:
      id-token: write
```
2. Run super-linter against the workflow - an error will occur indicating the 'id-token' permission scope is unknown.

## Logs

.github/workflows/build.yml:23:7: unknown permission scope "id-token". all available permission scopes are "actions", "checks", "contents", "deployments", "issues", "metadata", "packages", "pull-requests", "repository-projects", "security-events", "statuses" [permissions]

## Additional context

Fixed in this actionlint PR: https://github.com/rhysd/actionlint/pull/62

**Comment by mike-stewart** [comment]
This is now resolved.

**Comment by ferrarimarco** [comment]
We now ship version 1.6.8.
